Sandbox/src/SandboxApp.cpp

```diff
@@ -1,210 +1,16 @@
-			#version 330 core
-			
-			layout(location = 0) in vec3 a_Position;
-
-			uniform mat4 u_ViewProjection;
-			uniform mat4 u_Transform;
-
-			out vec3 v_Position;
-
-			void main()
-			{
-				v_Position = a_Position;
-				gl_Position = u_ViewProjection * u_Transform * vec4(a_Position, 1.0);	
-			}
-		)";
-
-		std::string flatColorShaderFragmentSrc = R"(
-			#version 330 core
-			
-			layout(location = 0) out vec4 color;
-
-			in vec3 v_Position;
-			
-			uniform vec3 u_Color;
-
-			void main()
-			{
-				color = vec4(u_Color, 1.0);
-			}
-		)";
-
-		m_FlatColorShader = Hazel::Shader::Create("FlatColor", flatColorShaderVertexSrc, flatColorShaderFragmentSrc);
-
-		auto textureShader = m_ShaderLibrary.Load("assets/shaders/Texture.glsl");
-
-		m_Texture = Hazel::Texture2D::Create("assets/textures/Checkerboard.png");
-		m_ChernoLogoTexture = Hazel::Texture2D::Create("assets/textures/ChernoLogo.png");
-
-		textureShader->Bind();
-		textureShader->SetInt("u_Texture", 0);
-	}
-
-	void OnUpdate(Hazel::Timestep ts) override
-	{
-		// Update
-		m_CameraController.OnUpdate(ts);
-
-		// Render
-		Hazel::RenderCommand::SetClearColor({ 0.1f, 0.1f, 0.1f, 1 });
-		Hazel::RenderCommand::Clear();
-
-		Hazel::Renderer::BeginScene(m_CameraController.GetCamera());
-
-		glm::mat4 scale = glm::scale(glm::mat4(1.0f), glm::vec3(0.1f));
-
-		m_FlatColorShader->Bind();
-		m_FlatColorShader->SetFloat3("u_Color", m_SquareColor);
-
-		for (int y = 0; y < 20; y++)
-		{
-			for (int x = 0; x < 20; x++)
-			{
-				glm::vec3 pos(x * 0.11f, y * 0.11f, 0.0f);
-				glm::mat4 transform = glm::translate(glm::mat4(1.0f), pos) * scale;
-				Hazel::Renderer::Submit(m_FlatColorShader, m_SquareVA, transform);
-			}
-		}
-
-		auto textureShader = m_ShaderLibrary.Get("Texture");
-
-		m_Texture->Bind();
-		Hazel::Renderer::Submit(textureShader, m_SquareVA, glm::scale(glm::mat4(1.0f), glm::vec3(1.5f)));
-		m_ChernoLogoTexture->Bind();
-		Hazel::Renderer::Submit(textureShader, m_SquareVA, glm::scale(glm::mat4(1.0f), glm::vec3(1.5f)));
-
-		// Triangle
-		// Hazel::Renderer::Submit(m_Shader, m_VertexArray);
-
-		Hazel::Renderer::EndScene();
-	}
-
-	virtual void OnImGuiRender() override
-	{
-		ImGui::Begin("Settings");
-		ImGui::ColorEdit3("Square Color", glm::value_ptr(m_SquareColor));
-		ImGui::End();
-	}
-
-	void OnEvent(Hazel::Event& e) override
-	{
-		m_CameraController.OnEvent(e);
-	}
-private:
-	Hazel::ShaderLibrary m_ShaderLibrary;
-	Hazel::Ref<Hazel::Shader> m_Shader;
-	Hazel::Ref<Hazel::VertexArray> m_VertexArray;
-
-	Hazel::Ref<Hazel::Shader> m_FlatColorShader;
-	Hazel::Ref<Hazel::VertexArray> m_SquareVA;
-
-	Hazel::Ref<Hazel::Texture2D> m_Texture, m_ChernoLogoTexture;
-
-	Hazel::OrthographicCameraController m_CameraController;
-	glm::vec3 m_SquareColor = { 0.2f, 0.3f, 0.8f };
-};
+#include "ExampleLayer.h"
 
 class Sandbox : public Hazel::Application
 {
 public:
 	Sandbox()
 	{
-		// PushLayer(new ExampleLayer());
-		PushLayer(new Sandbox2D());
+		PushLayer(new ExampleLayer());
```

Comment:
One last thing, you should have the ExampleLayer commented out since when Cherno pulls next time he needs to be on ``Sandbox2D`` since that is what the series is currently on, it will make it easier for Cherno so he doesn't get confused on a video. Didn't notice this until now. 😛 

You should just uncomment Sandbox2D and have ExampleLayer commented out.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/170/commits/be8290d630af3aef401ae3a6b1fa46088398675d

Reply:
https://github.com/TheCherno/Hazel/pull/170/commits/be8290d630af3aef401ae3a6b1fa46088398675d

Reply:
I am so sorry about that. When I updated the PR with upstream changes, I tested both layers again, but I forgot to return to the previous layer state after testing. Good catch!